justfile

```diff
@@ -25,15 +25,17 @@ build-binary VERSION=version:
     cd ./cmd/costmodel && \
         {{commonenv}} GOOS=linux GOARCH=amd64 go build \
         -ldflags \
-          "-X github.com/opencost/opencost/core/pkg/version.Version={{VERSION}} \
-           -X github.com/opencost/opencost/core/pkg/version.GitCommit={{commit}}" \
+          "-X github.com/opencost/opencost/pkg/version.Version={{VERSION}} \
+           -X github.com/opencost/opencost/pkg/version.Architecture=amd64 \
+           -X github.com/opencost/opencost/pkg/version.GitCommit={{commit}}" \
         -o ./costmodel-amd64
 
     cd ./cmd/costmodel && \
         {{commonenv}} GOOS=linux GOARCH=arm64 go build \
         -ldflags \
-          "-X github.com/opencost/opencost/core/pkg/version.Version={{VERSION}} \
-           -X github.com/opencost/opencost/core/pkg/version.GitCommit={{commit}}" \
+          "-X github.com/opencost/opencost/pkg/version.Version={{VERSION}} \
+           -X github.com/opencost/opencost/pkg/version.Architecture=arm64 \
```

Comment:
we can revert these changes

Reply:
Agreed, we don't need `version.Architecture` anymore.

Reply:
gone